api/src/test/java/org/openmrs/module/ModuleFileParserTest.java

```diff
@@ -157,17 +151,17 @@ public void parse_shouldParseValidLogicModuleFromFile() {
 		assertThat(module.getMappingFiles(), hasItems("LogicRuleToken.hbm.xml"));
 	}
 
-	private void expectModuleExceptionWithTranslatedMessage(String s) {
+	/*private void expectModuleExceptionWit hTranslatedMessage(String s) {
```

Comment:
@achilep please in general do never just comment out methods. If your changes makes something fail you need to understand why that is. If in this case the method is not needed anymore delete it. Please delete the 2 commented methods.